<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDesktopTip</class>
 <widget class="QWidget" name="RDesktopTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{color: rgb(35,211,146);font:20px;}
/*纵轴整体设置*/
QScrollBar:vertical
{
    width:16px;
    background:rgb(41,97,84);
    margin:0px,0px,0px,0px;
    padding-top:16px;
    padding-bottom:16px;
}

/*纵轴-滚动条设置*/
QScrollBar::handle:vertical
{
    background:rgb(4,200,147);
    border-radius:1px;
    min-height:20px;
}

/*纵轴-触发滚动条-设置*/
QScrollBar::handle:vertical:hover
{
    background:rgb(4,250,150);
    border-radius:1px;
    min-height:20;
}

/*纵轴-上箭头-设置*/
QScrollBar::sub-line:vertical
{
    height:16px;width:16px;
        border-image: url(:/static/image/mainwindow/上箭头.png);
    subcontrol-position:top;
}

/*纵轴-触发上箭头-设置*/
QScrollBar::sub-line:vertical:hover
{
    height:16px;width:16px;
        border-image: url(:/static/image/mainwindow/上箭头-触发.png);
    subcontrol-position:top;
}

/*纵轴-下箭头-设置*/
QScrollBar::add-line:vertical 
{
    height:16px;width:16px;
        border-image: url(:/static/image/mainwindow/下箭头.png);
    subcontrol-position:bottom;
}

/*纵轴-触发下箭头-设置*/
QScrollBar::add-line:vertical:hover
{
    height:16px;width:16px;
        border-image: url(:/static/image/mainwindow/下箭头-触发.png);
    subcontrol-position:bottom;
}

/*纵轴-滚动条底色设置*/
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgb(41,97,84);
    border-radius:0px;
}
/*横轴整体设置*/
QScrollBar:horizontal
{
    height:16px;
    background:rgb(41,97,84);
    margin:0px,0px,0px,0px;
    padding-left:16px;
    padding-right:16px;
}

/*横轴-滚动条设置*/
QScrollBar::handle:horizontal
{
        height:16px;
    background:rgb(4,200,147);
    border-radius:1px;
}

/*横轴-触发滚动条-设置*/
QScrollBar::handle:horizontal:hover
{
        height:16px;
    background:rgb(4,250,150);
    border-radius:1px;
}

/*横轴-左箭头-设置*/
QScrollBar::sub-line:horizontal
{
    height:16px;width:16px;
        border-image: url(:/static/image/mainwindow/左箭头.png);
    subcontrol-position:left;
}

/*横轴-触发左箭头-设置*/
QScrollBar::sub-line:horizontal:hover
{
    height:16px;width:16px;
    border-image: url(:/static/image/mainwindow/左箭头-触发.png);
    subcontrol-position:left;
}

/*横轴-右箭头-设置*/
QScrollBar::add-line:horizontal 
{
    height:16px;width:16px;
    border-image: url(:/static/image/mainwindow/右箭头.png);
    subcontrol-position:right;
}

/*横轴-触发右箭头-设置*/
QScrollBar::add-line:horizontal:hover
{
    height:16px;width:16px;
    border-image: url(:/static/image/mainwindow/右箭头-触发.png);
    subcontrol-position:right;
}

/*横轴-滚动条底色设置*/
QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal
{
    background:rgb(41,97,84);
    border-radius:0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(16, 100, 68);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35,211,146);
</string>
        </property>
        <property name="text">
         <string> 信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: rgb(35,211,146);background-color: rgb(16, 100, 68);border:none;}

</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 36, 36);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">
border:none;
background-color: rgb(0,36,36);
color:rgb(35,211,146);
</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>507</width>
           <height>295</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">/*#scrollAreaWidgetContents{border-image: url(:/static/image/mainwindow/修改版弹窗.png);}*/</string>
         </property>
         <layout class="QVBoxLayout" name="layout_labels" stretch="100">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
